class.jetpack-jitm.php

```diff
@@ -21,12 +21,53 @@ static function init() {
 	}
 
 	private function __construct() {
+		if ( ! Jetpack::is_active() ) {
```

Comment:
Might make sense to add an early check for `current_user_can( 'jetpack_manage_modules' )` as well.